recaptcha_enterprise/demosite/src/main/java/app/MainController.java

```diff
@@ -358,4 +366,33 @@ public static ModelAndView comment() {
     }
   }
 
+  // Classify the action as BAD/ NOT_BAD based on conditions specified.
+  public static HashMap<String, String> checkForBadAction(Assessment assessmentResponse, String recaptchaAction) {
+    String reason = "";
+    String label = Label.NOT_BAD.getLabel();
+    HashMap<String, String> result = new HashMap<>();
+
+    // Check if the token obtained from client is valid.
+    if (!assessmentResponse.getTokenProperties().getValid()) {
+      reason = Error.INVALID_TOKEN.getErrorMessage();
+      label = Label.BAD.getLabel();
+    }
+
+    // Check if the returned recaptcha action matches the expected.
+    if (!assessmentResponse.getTokenProperties().getAction().equals(recaptchaAction)) {
+      reason = Error.ACTION_MISMATCH.getErrorMessage();
+      label = Label.BAD.getLabel();
+    }
+
+    // Check if the returned score is greater less than or equal to the threshold set.
```

Comment:
"greater less than or equal to" -> "greater than"

Reply:
Updated it in previous commit. Thanks